partner_invoicing_mode/tests/common.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2020 Camptocamp SA
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.tests.common import TransactionCase
+
+
+class CommonPartnerInvoicingMode(TransactionCase):
```

Comment:
Please replace by the `BaseCommon`
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-18.0#:~:text=Use%20BaseCommon%20as%20base%20test%20class%20to%20reduce%20overhead%20from%20tracking%20and%20boost%20the%20test%20suite